lib-gdal-preferences-ui/src/main/java/org/esa/s2tbx/dataio/gdal/preferences/GDALOptionsPanel.java

```diff
@@ -0,0 +1,165 @@
+package org.esa.s2tbx.dataio.gdal.preferences;
```

Comment:
we should replace org.esa.s2tbx with org.esa.snap

Reply:
Updates made.